docs/releases/v3.1.1.md

```diff
@@ -19,15 +18,42 @@ bases:
     version: v3.1.1
 ```
 
-2. Download the module:
+2.Download the module:
 
 ```bash
 furyctl legacy vendor
 ```
 
-3. Build your desired profile Kustomize base and apply it to the cluster, for example for the `minimal` profile run:
+3.Build your desired profile Kustomize base and apply it to the cluster, for example for the `minimal` profile run:
 
 ```bash
 kustomize build /vendor/katalog/istio/istio-operator/profiles/minimal | kubectl apply -f
 ```
 
+Jaeger Operator Upgrade Notes
+
+In this release, Jaeger has been updated to version 1.61.0. If you are using the Jaeger operator as part of your Istio deployment, please note the following updates and distinctions between the Jaeger Operator and the Jaeger Operated components.
+
+Jaeger Operator
+
+1.No Special Action for Jaeger Operator Required: The Jaeger operator itself does not require any special upgrade steps. You can apply the project as usual, and it will continue to function without requiring any manual intervention or changes to the operator.
+
+To apply the latest version of the Jaeger operator, run the following command:
+
+``` bash
+kustomize build /vendor/katalog/istio-operator/jaeger-operator | kubectl apply -f
+```
+
+Jaeger Operated
+
+1.Namespace Standardization for Jaeger Operated: The namespace used by the Jaeger Operated component has been updated to follow the standard conventions used by the Jaeger operator. This change ensures consistency across deployments. If you are upgrading from a previous version, review any custom namespace configurations to ensure compatibility with this update.
```

Comment:
```suggestion
1. Namespace Standardization for Jaeger Operated: The namespace used by the Jaeger Operated component has been updated from `jaeger` to `observability` to follow the standard conventions used by the Jaeger operator. This change ensures consistency across deployments. If you are upgrading from a previous version, review any custom namespace configurations to ensure compatibility with this update.
```

what do you mean by "custom namespace configurations"? If I deployed the previous version of the module as-is (that used a different namespace) than this release, does it count as a custom namespace configuration?